Cargo.toml

```diff
@@ -178,6 +178,7 @@ default = [
   "wayland",
   "debug",
   "zstd_rust",
+  "bevy_dev_tools",
```

Comment:
I'm of the mind that this is better suited to some "dev" cargo feature (which we've discussed in the past), which would enable a common collection of features useful for development: tracing, dev tools, asset hot reloading, embedded hot reloading, etc

Reply:
In general I think we should consider moving in the direction of a smaller baseline feature set with larger "meta" features that pull in targeted collections of cargo features. I consider `--no-default-features` to be a non-viable lifestyle for people actually building apps.